### DISCUSSION TITLE
Does any specific development demo or documents for beginners?

### DISCUSSION THREAD

**Discussion by Kendr1ckFu** [yomorun/yomo#264] [General] [answered]
Heeey, I'm currently a student, and now I'm just starting to learn the IoT and QUIC protocols, I found this open source project when I visited Github, it's great, so I wanna use it to develop a demo project, but I found that there is  little documentation for novices, so I asked if there is a small demo for my newbies and standardized documentation, thank you soooo much!

**Comment by xiaojian-hong** [comment, marked as answer]
Hi @fukaicheng ,
Thanks for interesting the YoMo project.
You can find the basic example for beginner on https://github.com/yomorun/yomo/tree/master/example/basic.

**Other resources**
- YoMo documentation: https://docs.yomo.run/ (English & Chinese)
- Awesome QUIC: https://github.com/10cella/awesome-quic (English & Chinese)
- IoT example: https://github.com/yomorun/example-noise (Chinese)
  - hardware: https://docs.yomo.run/zh/hardware

**Reply by Kendr1ckFu** [reply]
Ohh, thank u very much for ur prompt answer.
I think I need to spend some time looking over these documents and demos to get started. 
Also hope that the YoMo framework will be discovered and used by more people.👏👏👏

**Comment by fanweixiao** [comment]
Hi @fukaicheng, glad to see you here :) in the last two years there are students using YoMo to implement their graduation design. if you need some discussion about your project, you can email me cc@yomo.run.

**Reply by Kendr1ckFu** [reply]
Hello! I am honored to receive your reply 😀😀.
As a matter of fact, I am also preparing for my graduation design, which is a new challenge for me, because I used to be exposed to the content related to machine learning 🧐.
If I have any questions to bother you in the future, please give me your advice. Thank you again 🙏🙏.
